src/hooks/usePermit2Allowance.ts

```diff
@@ -70,7 +70,7 @@ export default function usePermit2Allowance(amount?: CurrencyAmount<Token>, spen
   // Signature and PermitAllowance will expire, so they should be rechecked at an interval.
   const [now, setNow] = useState(Date.now())
   // Calculate now such that the signature will still be valid for the submitting block.
-  useInterval(() => setNow((Date.now() - STANDARD_L1_BLOCK_TIME) / 1000), STANDARD_L1_BLOCK_TIME, true)
+  useInterval(() => setNow((Date.now() + STANDARD_L1_BLOCK_TIME) / 1000), STANDARD_L1_BLOCK_TIME, true)
```

Comment:
what's going on with this change?

Reply:
It's the one bug I caught in my tests 🤣 

"still be valid for the submitting block" means we need to pretend we're in the future, not in the past.

Reply:
huge, worth it then